jupyterhub/jupyterhub/base/jupyterhub-configmap.yaml

```diff
@@ -5,7 +5,26 @@ metadata:
     app: jupyterhub
   name: jupyterhub-cfg
 data:
-  jupyterhub_config.py: ""
+  jupyterhub_config.py: |
```

Comment:
This should be part of `jupyterhub-odh` repo - we should keep this CM clean for admin/user customizations

Reply:
ack

Reply:
Updated the MR to jupyterhub-odh to reflect this change: https://github.com/opendatahub-io/jupyterhub-odh/pull/104